### ISSUE TITLE
clients_ingress_sg role

### ISSUE THREAD

**Issue by vitbyst** [hashicorp/learn-nomad-cluster-setup#4]
what is `resource 1"aws_security_group" "clients_ingress_sg"` for?

**Comment by shankarnarayanb** [comment]
I agree, `clients_ingress_sg ` security group is not being directly referenced in the aws_instance resource blocks in the `main.tf` and could be potentially removed. Looks like @tunzor orginally intended this to be used to allow incoming traffic from the Internet to the client instances within an aws_autoscaling_group but I am not really sure? @tunzor, would you be happy for me to raise a PR to make the change to remove `clients_ingress_sg`?

**Comment by tunzor** [comment]
Thanks for the comments! @shankarnarayanb is correct, it was intended for ingress to applications running on the client nodes. 

We've got some updates coming soon that include the renaming of resources for clarity so you don't need to open a PR but it's very much appreciated!
